sync_packages.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+robot_name="${HOSTNAME//-b1}"
+
+if [ "$HOSTNAME" != "$robot_name-b1" ]; then 
+	echo "FATAL: CAN ONLY BE EXECUTED ON BASE PC"
+	exit
+fi
+
+sudo apt list --installed > /tmp/package_list
+perl -pi -e 's{ / .*? ]}{}x' /tmp/package_list
+sed -i 's/Listing...//' /tmp/package_list 
+sed -i ':a;N;$!ba;s/\n/ /g' /tmp/package_list
+tr '\n' ' ' < /tmp/package_list | sed -i '$s/ $/\n/' /tmp/package_list
+packages=$(cat /tmp/package_list)
+
+pcs="
+$robot_name-b1
+$robot_name-t1
+$robot_name-t2
+$robot_name-t3
+$robot_name-s1
+$robot_name-h1"
```

Comment:
they had last week a problem with missed packages on one PC of the robot, I though that they could use the functionality of this script to avoid this kind of problems... but all the software/documentation of this repository is specific for cob4. The majority of the scripts are only useful for a robot with 3 scanners, flexisoft, kinects, mimic, sound, hands with a bluetooth connection...